doc/source/timeseries.rst

```diff
@@ -191,6 +187,25 @@ or ``format``, use ``to_datetime`` if these are required.
 
     pd.Timestamp('2010/11/12')
 
```

Comment:
can you add a section ref tag here as well.

Reply:
I have put a reference to the `pd.Timestamp` documentation as above and pushed.